src/badguy/badguy.hpp

```diff
@@ -49,8 +49,15 @@ class BadGuy : public MovingSprite,
       state and calls active_update and inactive_update */
   virtual void update(float dt_sec) override;
 
-  virtual std::string get_class() const override { return "badguy"; }
-  virtual std::string get_display_name() const override { return _("Badguy"); }
+  virtual std::string get_class() const override
+  {
+    return "badguy";
+  }
+
+  virtual std::string get_display_name() const override
+  {
+    return _("Badguy");
+  }
```

Comment:
Lines like these in header files are made to be written on a single line. It's because they are trivial and are not expected to require any maintenance in the foreseeable future.

Reply:
Fixed

Reply:
I'm marking this as unresolved, there are multiple other files with similar problems. The badguy wasn't the only one :stuck_out_tongue: 

Would you prefer me to mark each place in individual reviews so you can mark them as resolved progressively?

Reply:
Sure